### PR TITLE
Cartopy-dep

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Fix: cartopy WMS](https://trello.com/c/pImtg5PS/1163-fix-cartopy-wms)

File: examples/wms.ipynb [hunk 1]
```diff
@@ -142,10 +160,11 @@
     "ax = plt.axes(projection=ccrs.PlateCarree())\n",
     "\n",
     "ax.add_wms(wms,\n",
-    "           layers=[str(workflow)],\n",
+    "           layers=[str(workflow)],           \n",
     "           wms_kwargs={\n",
     "                'time': urllib.parse.quote(bbox.time_str),\n",
-    "                'crs': bbox.srs\n",
+    "                'crs': bbox.srs,\n",
+    "                'exceptions': 'application/json'\n",
```
Comment (jdroenner) [inline examples/wms.ipynb:167]: is this required or optional?

Comment (michaelmattig) [reply examples/wms.ipynb:167]: owslib sets it explicitly to XML, so I override it here.

Comment (jdroenner) [reply examples/wms.ipynb:167]: i guess we should we set that on a global level for wms requests to the geo engine?

Comment (michaelmattig) [reply examples/wms.ipynb]: I added support for XML in the backend and removed the format here c16e108fc21705b63fcdfad3645128e4b669a0f3

**Review by jdroenner — APPROVED** [review]

**Comment by jdroenner** [comment]
bors merge

**Pull request merged**